client/src/components/common/boost/BoostConfigurator.tsx

```diff
@@ -112,6 +177,13 @@ export default function BoostConfigurator({
         <Card.Body>
           <Card.Title style={{ marginBottom: '1.5rem' }}>
             Configuration
+            <Button
+              variant="primary"
+              className="mx-3 float-right"
```

Comment:
Try `ml-3` - IMO it looks a bit nicer, as the button is aligned with the UI below it.